O/OpenBLAS/OpenBLAS32@0.3.13/bundled/patches/dasum_microk_skylakex-typecast.patch

```diff
@@ -0,0 +1,17 @@
+--- a/kernel/x86_64/dasum_microk_skylakex-2.c
++++ b/kernel/x86_64/dasum_microk_skylakex-2.c
+@@ -39,10 +39,10 @@
```

Comment:
Uhm, I believe this patch is wrong (I made it, so I have only to blame myself).  I need to double-check it later

Reply:
Yes, this is a [GCC bug](https://gcc.gnu.org/bugzilla/show_bug.cgi?id=87467).  The patch tried to accommodate the bugged signature of the function, which is wrong.  We should either [patch the header file on-the-fly](https://github.com/gcc-mirror/gcc/commit/23bce99cbe7016a04e14c2163ed3fe6a5a64f4e2#diff-c3bbd1dcd7c953ef6d719969fb187e31209ef3d7851a84448aad949aea2e8106) or require GCC v9, where this should be already fixed.  (I think waiting to fix all of our bugged shards is not really an option.)  I'll ask on Slack

Reply:
Should this patch be added to Julia for source builds of OpenBLAS?

Reply:
Not sure how that would work, I think you don't want (and most often can't) apply a patch to GCC header files.

Reply:
Ah, I totally missed that these were kernel patches

Reply:
Ok, the poll on Slack suggests to patch the header file `gcc/config/i386/avx512fintrin.h` of GCC.  This patch is definitely wrong and should be removed in any case.  In the future we'll likely backport the patch to all of our versions of GCC affected by the issue.

Reply:
I've removed this patch from OpenBLAS 0.3.13. Let me know if I should also remove it from the other versions of OpenBLAS

Reply:
Re-reading your comments again I see I was supposed to remote the skylakex patch and apply the GCC patch. I'm unaware of how this is done with BB so I'll just give it a shot and let the CI sort out if it's correct.

Reply:
I can have a look later today

Reply:
I've attempted to patch the avx512fintrin.h GCC header file. The patch applies cleanly on GCC 7.1.0 and 8.1.0 so I'm now attempting a CI run.

Reply:
I changed OpenBLAS 0.3.12 to patch the GCC header too